README.md

```diff
@@ -145,6 +145,16 @@ To expose internal logs to the output, set the following properties:
 }
 ```
 
+To specify a subfolder in which the react-native project is located, set this one:
+
+```
+{
+  "react-native-tools": {
+    "projectRoot": "your-react-native-project"
```

Comment:
This is a path, relative to workspace's root, right? Could you please make it more clear?